great_expectations/gx/plugins/expectations/expect_column_values_to_have_list_members.py

```diff
@@ -0,0 +1,145 @@
+from great_expectations.expectations.expectation import ColumnMapExpectation
+from great_expectations.expectations.metrics import (
+    ColumnMapMetricProvider,
+    column_condition_partial,
+)
+
+
+# This class defines a Metric to support your Expectation.
+# For most ColumnMapExpectations, the main business logic for calculation will live in this class.
+class ColumnValuesListMembers(ColumnMapMetricProvider):
+    """Class definition for list members checking metric."""
+
+    # This is the id string that will be used to reference your metric.
+    condition_metric_name = "column_values.list_members"
+    condition_value_keys = ("list_members",)
+
+    # This method implements the core logic for the PandasExecutionEngine
+    @column_condition_partial(engine=PandasExecutionEngine)
+    def _pandas(
+        cls, column: pd.core.series.Series, list_members: set, **kwargs
+    ) -> bool:
+        """Core logic for list member checking metric on a
+        pandas execution engine.
+
+        Args:
+            column (pd.core.series.Series): Pandas column to be evaluated.
+            list_members (set): Expected list members.
+        Returns:
+            bool: Whether or not the column values have the expected list members.
+        """
+        return column.apply(lambda x: cls._check_list_members(x, list_members))
+
+    @staticmethod
+    def _check_list_members(cell: Any, list_members: set) -> bool:
+        """Check if a cell is a list, and if it has the expected members.
+
+        Args:
+            cell (Any): Individual cell to be evaluated.
+            list_members (set): Expected list members.
+
+        Returns:
+            bool: Whether or not the cell is a list with the expected members.
+        """
+        if not isinstance(cell, list):
+            return False
+        if not all(item in list_members for item in cell):
+            return False
+        return True
+
+    # This method defines the business logic for evaluating your metric when using a SqlAlchemyExecutionEngine
+    # @column_condition_partial(engine=SqlAlchemyExecutionEngine)
+    # def _sqlalchemy(cls, column, _dialect, **kwargs):
+    #     raise NotImplementedError
+
+    # This method defines the business logic for evaluating your metric when using a SparkDFExecutionEngine
+    # @column_condition_partial(engine=SparkDFExecutionEngine)
+    # def _spark(cls, column, **kwargs):
+    #     raise NotImplementedError
+
+
+# This class defines the Expectation itself
+class ExpectColumnValuesToHaveListMembers(ColumnMapExpectation):
+    """Expect the list in column values to have certain members."""
+
+    # These examples will be shown in the public gallery.
+    # They will also be executed as unit tests for your Expectation.
+    examples = [
+        {
+            "data": {
+                "a": [["a", "b", "a", "b"]],
+                "b": [["a", "b", "c", "d"]],
+            },
+            "tests": [
```

Comment:
This is really awesome that the test itself is just contained.  i wish we could do something like that for DCQC but its way too complicated.